packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx

```diff
@@ -167,7 +167,7 @@ export const Mappings = () => {
             ) : mappings.length === 0 ? (
               <EmptyMessage message="No Mappings available." />
             ) : (
-              <div className = "mappings-list">
+              <div className = "gmw-mappings-list">
```

Comment:
Inconsistent formatting - drop the spaces surrounding `=`.

Reply:
Done